### ISSUE TITLE
Failures in full testsuite

### ISSUE THREAD

**Issue by johnaohara** [kruize/hpo#59]
As I was experimenting for https://github.com/kruize/hpo/issues/58 I noticed there a re a few failures when running the full testsuite, e.g. 

```
########### Results Summary of the test suite hpo_api_tests ##########
hpo_api_tests took 2562 seconds
Number of tests performed 56
Number of tests passed 42
Number of tests failed 14

~~~~~~~~~~~~~~~~~~~~~~~ hpo_api_tests failed ~~~~~~~~~~~~~~~~~~~~~~~~~~
Failed cases are :
		  invalid-id
		  empty-id
		  invalid-searchspace
		  empty-name
		  null-name
		  invalid-trial-result
		  empty-trial-result
		  null-trial-result
		  invalid-result-value-type
		  empty-result-value-type
		  null-result-value-type
		  invalid-result-value
		  null-result-value
		  additional-field
```

Is this something that someone is currently investigating?

**Comment by dinogun** [comment]
@chandrams should be looking into this

**Comment by khansaad** [comment]
This was fixed in PR #60
